draft-ietf-httpbis-h3-websockets.md

```diff
@@ -71,6 +71,10 @@ in HTTP/2 as defined {{!RFC8441}}. {{Appendix A.3 of HTTP3}} requires that
 HTTP/3 settings be registered separately for HTTP/3. The
 SETTINGS_ENABLE_CONNECT_PROTOCOL value is 0x08 (decimal 8), as in HTTP/2.
 
+If a server receives an Extended CONNECT request with a ":protocol"
+value that is unknown or is not supported, the server SHOULD respond
+to the request with a 501 status code, see {{Section 6.6.2 of RFC7231}}.
+
```

Comment:
@mnot suggested supplying a header with this error and that sounds good. But I'm not sure which one he intended. Proxy-Status would not seem right though

Reply:
Agreed. Hopefully @mnot can suggest an appropriate header here.

Reply:
Body sounds like a fine place for the explanation to me.

Re: Alt-Svc, that's a fair point. It makes me wonder if we should consider (in a different draft) adding an httpOnly= tag to Alt-Svc to restrict the advertisement to HTTP traffic, and not other protocols (or some similar mechanism) to avoid the potential problem @martinthomson describes?

Reply:
I don't think that is backwardly compatible.  Nor would a flag that says "just send this request (or requests to this resource) to the alternative".

Reply:
Body is fine; perhaps mention RFC7807 as an option.

Reply:
Thanks for the suggestion. Mentioned RFC 7807